<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExpenditure</class>
 <widget class="QDialog" name="DialogExpenditure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="activitiesLineEdit">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Put your activity here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="moneyLineEdit">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Put your money here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addPushBtn">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 247, 255);</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculatePushBtn">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 247, 255);</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removePushBtn">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 247, 255);</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 247, 255);</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 255, 242);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
